index.js

```diff
@@ -22,6 +22,8 @@ const authors = {
   mmckegg: 'https://www.patreon.com/MattMcKegg',
   moox: 'https://liberapay.com/MoOx/',
   mpj: 'https://www.patreon.com/funfunfunction',
+  mrahmadawais: 'https://pay.paddle.com/checkout/515568',
+  ahmadawais: 'https://pay.paddle.com/checkout/515568',
```

Comment:
This is not a valid npm user: https://www.npmjs.com/~ahmadawais

Please remove this line